x-pack/plugins/security/public/views/management/edit_user/edit_user.js

```diff
@@ -17,7 +17,7 @@ import { render, unmountComponentAtNode } from 'react-dom';
 import { I18nContext } from 'ui/i18n';
 import { getEditUserBreadcrumbs, getCreateUserBreadcrumbs } from '../breadcrumbs';
 
-const renderReact = (elem, httpClient, changeUrl, username) => {
+const renderReact = (elem, httpClient, changeUrl, username) => { // eslint-disable-line no-unused-vars
```

Comment:
I think we can remove the `httpClient` variable here, and then from the controller/`renderReact()` invocation below.